winetricks: add page (#5265)
tldr-pages/tldr: pages/linux/winetricks.md

# winetricks

> > Manage virtual Windows environments.

Reviewer: ```suggestion
> Manage Wine virtual Windows environments.
```

Author: I think that's a lot better than what reads in `man`: `winetricks - manage virtual Windows environments using Wine`

The "using Wine" is really vague.

> More information: <https://wiki.winehq.org/Winetricks>.

> - Start a graphical setup at the default wine location:

Reviewer: ```suggestion
- Start a graphical setup at the default Wine location:
```

`winetricks`

> - Specify a wine prefix to run winetricks in:

Reviewer: The term 'prefix' is quite common across our pages, but I think it conveys the point just as well to use 'directory'. 
```suggestion
- Specify a custom wine directory to run winetricks in:
```

> - Specify a custom wine directory to run winetricks in:

Reviewer: ```suggestion
- Specify a custom Wine directory to run winetricks in:
```

> `WINEPREFIX={{/path/to/wine/prefix}} winetricks`

Reviewer: ```suggestion
`WINEPREFIX={{path/to/wine/directory}} winetricks`
```

Author: Can the WINEPREFIX actually be relative? Just asking because I don't know.

Reviewer: I have no idea, it's just what we always do.  We should discuss adding this to the contributing guidelines sometime...:thinking:

Reviewer 2: Based on the Wiki I have it can be relative:
```bas
env WINEPREFIX=~/.winetest sh winetricks mfc40 
```

about a page with a command that requires a absolute path, we've an example here: https://github.com/tldr-pages/tldr/blob/master/pages/linux/ksvgtopng5.md

> `WINEPREFIX={{path/to/wine/directory}} winetricks`

Reviewer 2: I think `wine_directory` is easier to read:
```suggestion
`WINEPREFIX={{path/to/wine_directory}} winetricks`
```

> - Install a Windows DDL or a component to the default prefix:

Reviewer: ```suggestion
- Install a Windows DLL or a component to the default prefix:
```

> - Install a Windows DLL or a component to the default prefix:

Reviewer: ```suggestion
- Install a Windows DLL or a component to the default directory:
```

> - Install a Windows DLL or component to the default directory:

Reviewer: ```suggestion
- Install a Windows DLL or component to the default Wine directory:
```

> - Install a Windows DLL or a component to the default directory:

Reviewer: By 'a component' do mean 'Windows component', like 'Windows DLL'?  If so, we can remove 'a':
```suggestion
- Install a Windows DLL or component to the default directory:
```
